apps/website/src/theme/Navbar/Layout/index.tsx

```diff
@@ -1,12 +1,12 @@
 import { useThemeConfig } from "@docusaurus/theme-common";
-import {
+const {
   useHideableNavbar,
   useNavbarMobileSidebar,
-} from "@docusaurus/theme-common/internal";
+} = require("@docusaurus/theme-common/internal");
```

Comment:
docusaurus does not expose types for `/internal` needed to require them.

Reply:
I'd keep it as an import, and just @ts-expect-error it